gen/armor.py

```diff
@@ -545,6 +546,14 @@ def plan_action_for_groups(
                 dcs_group.add_waypoint(retreat_point, PointAction.OffRoad)
                 dcs_group.add_waypoint(reposition_point, PointAction.OffRoad)
 
+            if len(dcs_group.points) >= 2:
+                try:
+                    dcs_group.points[1].ETA = from_cp.delayed_start.total_seconds()
+                    dcs_group.points[1].ETA_locked = True
+                    dcs_group.points[1].speed_locked = False
```

Comment:
The fact that WP1 _must_ be close to WP0 for this to work is a really easy thing to overlook for anyone that might be altering front line behavior. Instead, create a wait action with an end time like we do for hold points for air units?